synapse/rest/client/v2_alpha/account_data.py

```diff
@@ -82,6 +82,13 @@ def on_PUT(self, request, user_id, room_id, account_data_type):
 
         body = parse_json_object_from_request(request)
 
+        if account_data_type == "m.read_marker":
+            raise SynapseError(
+                405,
+                "Cannot set m.read_marker through this API. "
+                "Use /rooms/!roomId:server.name/read_marker"
```

Comment:
For future reference we tend to put spaces at the start:

```python
"Cannot set m.read_marker through this API."
" Use /rooms/!roomId:server.name/read_marker"
```

This makes it more obvious that there is at least one space, which is quite important for e.g. SQL